source/2020-04-17-the-ember-times-issue-144.md

```diff
@@ -0,0 +1,151 @@
+---
+title: The Ember Times - Issue No. 144
+author: Chris Ng, Jessica Jordan, the crowd
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
+alias : "blog/2020/04/17-the-ember-times-issue-144.html"
+responsive: true
+---
+
+<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
+
+<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Quest Issue: Drop Mixins from ember-simple-auth 🔐,
+Call for a11y-focussed contributors ❤️,
+READMORE
+
+---
+
+## [Call for contributors: Join the strike team for accessibility in Ember](https://github.com/emberjs/rfcs/issues/595)
+
+The Ember community put in lots of work into [framework features](https://guides.emberjs.com/release/accessibility/) and [addons](https://emberobserver.com/categories/accessibility) to allow us to build accessible and user-friendly applications. But still there is more work to be done to **address accessibility issues** in Ember.js and to improve the future of how we build JavaScript apps.
+
+Do you want to be part of these efforts to help modern Ember apps become accessible?
+
+Then be sure join the **#st-a11y strike team** on the [Ember Discord chat](https://discordapp.com/invite/emberjs)! The recently founded strike team for creating an **accessible JavaScript framework** has already put together a [detailed list of issues to work on](https://github.com/emberjs/rfcs/issues/595) and is counting on your support. If you have the time and interest to help out, [join the team today](https://discordapp.com/invite/emberjs)!
```

Comment:
```suggestion
Then be sure join the **#st-a11y strike team** on the [Ember Discord chat](https://discordapp.com/invite/emberjs)! The recently founded strike team for creating an **accessible JavaScript framework** has already put together a [detailed list of issues to work on](https://github.com/emberjs/rfcs/issues/595) and is counting on your support. If you have the time and interest to help out, [join the team today](https://discord.gg/emberjs)!
```

Reply:
just noticed this, was out of date, so manually made the change myself